frontend/lib/laletterbuilder/letter-builder/send-options.tsx

```diff
@@ -44,39 +51,97 @@ export const LaLetterBuilderSendOptions = MiddleProgressStep((props) => {
               )}
               label={li18n._(t`Select a mailing method`)}
             />
-            <p>
-              <b>
-                <Trans>Send for me</Trans>
-              </b>
-              <br />
-              <Trans>
-                We'll send your letter for you via certified mail in 1-2
-                business days, at no cost to you.
-              </Trans>
-            </p>
-            <hr />
-            <p>
-              <b>
-                <Trans>Download and send myself</Trans>
-              </b>
-            </p>
-            <p>
-              <Trans>
-                Not sure yet? If you need more time to decide, you can always
-                come back later. We've saved your work.
-              </Trans>
-            </p>
             <TextualFormField
               type="email"
               {...ctx.fieldPropsFor("email")}
               label={optionalizeLabel(
                 li18n._(t`Landlord/management company's email`)
               )}
             />
-            <ProgressButtons back={props.prevStep} isLoading={ctx.isLoading} />{" "}
+            <ProgressButtons back={props.prevStep} isLoading={ctx.isLoading} />
           </>
         )}
       </SessionUpdatingFormSubmitter>
+      <Route
+        path={LaLetterBuilderRouteInfo.locale.habitability.sendConfirmModal}
+        render={() => <ConfirmModal {...props} />}
+      />
     </Page>
   );
 });
+
+export const ConfirmModal: React.FC<{
+  nextStep: string;
+}> = (props) => {
+  const { session } = useContext(AppContext);
+  const userWillMail =
```

Comment:
Should just add a note here to make the habitability bit generalizable to other letter types